frontend/src/components/molecules/MatchItem/MatchItem.tsx

```diff
@@ -184,3 +185,28 @@ export const MatchItem = (props: MatchItemProps,) => {
    )
 };
 
+const TeamHoverButtonComponent = (props: {
+   matchId: string | undefined,
+   teamId: string | undefined,
+   firstPlayerName: string | undefined,
+   secondPlayerName: string | undefined,
+   agreeDialogCallback: (matchId: string, teamId: string) => void
+}) => {
+   const matchId = props.matchId;
+   const teamId = props.teamId;
+   const firstPlayerName = props.firstPlayerName;
+   const secondPlayerName = props.secondPlayerName;
+
+   if (!matchId || !teamId) return <div/>;
+
+   return (
+      <TeamHoverButton>
+         <AddCircleTwoToneIcon fontSize="inherit"/>
+         <MatchWinnerDeterminationDialog
+            agreeCallback={() => props.agreeDialogCallback(matchId, teamId)}
```

Comment:
chyba niepotrzebnie tutaj słowa callback - jak dobrze nazwiesz to będzie wiadomo :-) 
Jak jest ok i Tobie to pomaga to zostaw, ale jak wszędzie byśmy wrzucali dodatkowo callback do nazwy to zaraz byśmy dużo callbacków mieli :-)

Reply:
Jaką nazwę porponujesz na funkcę wywoływaną przy zamknięciu okna?